### PR TITLE
chore: update continue text and quotes

### PR DESCRIPTION
**Pull request by AashishDhakal**

### CODE REVIEW

**Review by marcbelmont — COMMENTED** [review]
Do you have ideas to make this code simpler?

File: docker/platerec_installer.py [hunk 1]
```diff
@@ -864,18 +719,10 @@ def uninstall_button_snapshot(n_clicks, hardware):
...
-        stop_container(STREAM_IMAGE)
-        container_id = get_container_id(STREAM_IMAGE)
-        if container_id:
-            cmd = f'docker container rm {container_id}'
-            os.system(cmd)
-        cmd = f'docker rmi "{STREAM_IMAGE}" -f'
-        os.system(cmd)
-        cmd = 'docker image prune -f'
-        os.system(cmd)
-        return [None, 'Image successfully uninstalled.']
```
Comment (AashishDhakal) [inline docker/platerec_installer.py:878]: @marcbelmont this was a duplicate, so transferred it to helper function.

**Comment by marcbelmont** [comment]
@AashishDhakal did you try building the executable like on https://github.com/parkpow/deep-license-plate-recognition/blob/master/.github/workflows/installers.yml

**Comment by AashishDhakal** [comment]
@marcbelmont yes did for mac.

**Review by marcbelmont — APPROVED** [review]
lgtm. Just make sure pyinstaller still works.

**Pull request merged**